src/operator/leaky_relu-inl.h

```diff
@@ -479,6 +479,18 @@ void LeakyReLUGradCompute(const nnvm::NodeAttrs& attrs,
   });
 }
 
+void DNNLLeakyReluForward(const nnvm::NodeAttrs& attrs,
+                          const OpContext& ctx,
+                          const NDArray& in_data,
+                          const OpReqType& req,
+                          const NDArray& out_data);
+
+void DNNLLeakyReluBackward(const nnvm::NodeAttrs& attrs,
+                           const OpContext& ctx,
+                           const std::vector<NDArray>& inputs,
+                           const std::vector<OpReqType>& req,
+                           const std::vector<NDArray>& outputs);
+
```

Comment:
I think it would be better to move these declaration to one of dnnl file - e.g. dnnl_activation-inl.h and then included in this file with #if MXNET_USE_ONEDNN == 1 guard
This applies to all other changes in this PR as well

btw. current declaration should be also guarded with #if MXNET_USE_ONEDNN == 1